### ISSUE TITLE
Undefined installedVersion causes report to fail

### ISSUE THREAD

**Issue by DanielStout5** [BePo65/license-report-recursive#27]
This block of code in the alphaSort function in util.js is causing generating the report to fail:

```
const aVersion = elementA.installedVersion;
const bVersion = elementB.installedVersion;

if (semver.lt(aVersion, bVersion)) {
  return -1;
}
if (semver.gt(aVersion, bVersion)) {
  return 1;
}
```

I'm not sure why but the "minimist" package has an undefined installedVersion for me, which causes `semver.lt` to fail.

**Comment by BePo65** [comment]
Hmm, I see - very strange (my e2e tests should have shown this effect). I'll take care of it at once :-)

**Comment by DanielStout5** [comment]
Thanks!

Another related error in the same spot after I added a temporary patch to skip the installedVersion comparison for undefined versions: `TypeError: Invalid Version: n/a`

So perhaps the check should be to check if the installedVersions are valid semver versions since apparently arbitrary values can be in there

**Comment by BePo65** [comment]
As far as my analysis by now, the problem is caused by a missing call to add data from the local package.json of the installed packages to the structure. I could reproduce the error and I think that over  the weekend I will have the time to fix it.

**Comment by BePo65** [comment]
Fixed with v6.4.1
